lib/protobuf/new_transform.ex

```diff
@@ -0,0 +1,15 @@
+defmodule Protobuf.NewTransform do
```

Comment:
Let's call this `Protobuf.TransformModule.InferFieldsFromEnum`, wdyt? It's a mouthful, but it's clear in what it does (also I would love for people not to use this as much as possible 😛).